docs/index.rst

```diff
@@ -1,3 +1,8 @@
+.. PyEPR documentation master file, created by
+   sphinx-quickstart on Wed May  6 22:02:11 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
```

Comment:
I think we can remove these comments - if you want to keep them, PyEPR -> PyERFA

Reply:
OK, will remove.
thanks